README.rst

```diff
@@ -9,8 +9,5 @@ Improvements compared to upstream zipfile stdlib:
 * Handling of symlinks (read and write)
 * Compatible 2.6 onwards (including 3.x), include context manager
 * Raises an exception by default when duplicate members are detected.
-
-Incoming:
-
-* Special class to limit number of members in an archive when unzipping,
-  to avoid DDos attacks using dummy zipfiles.
+* Special class `LeanZipFile` to avoid using too much memory when looking
+  into a zipfile with lots of archives.
```

Comment:
It should also have lower memory footprint then `ZipFile` on the same archive; it doesn't build and keep the whole index in memory but only extracts selected members of the archive.